exercises/practice/anagram/.docs/instructions.append.md

```diff
@@ -1,11 +1,7 @@
 # Hints
```

Comment:
BTW, it is possible to have a `.docs/hints.md` file that contains, well, hints.

Reply:
Good idea! I moved part of it over there.